ors-engine/src/main/resources/resources/locales/ors_da-DK.resources

```diff
@@ -0,0 +1,107 @@
+{
+    "instructions": {
+        "numerals": {
+            "1": "1.",
+            "2": "2.",
+            "3": "3.",
+            "4": "4.",
+            "5": "5.",
+            "6": "6.",
+            "7": "7.",
+            "8": "8.",
+            "9": "9.",
+            "10": "10."
```

Comment:
In Danish, there isn't a specific ruleset for when you written-out ordinal numbers. In Danish, the German _erste_ would be _første_, but it can also be written as _1._

Although as a rule of thumb in Danish you should use the written-out ordinal numbers as longs as it's ten or below. 

When I made the PR, I wasn't sure if there were any preferences to the numbers. I'll happily change it to be written out.

Reply:
As you want, either is fine I think :)

Reply:
I think, that the way it currently is would make more sense, if you were to use the directions for UI purposes.